ui/components/Navigator.js

```diff
@@ -315,6 +315,7 @@ const categories = [
     title : "Configuration",
     show : false,
     link : true,
+    isBeta : true,
```

Comment:
This can be removed, as there's no separate page for Configuration

Reply:
Yes, will remove it.